apps/backend/src/resolvers.ts

```diff
@@ -1,9 +1,24 @@
-import { getAllPosts } from "./services/post";
+import { createPost, deletePost, getAllPosts, getPostById } from "./services/post";
+import { Config } from "apollo-server";
 
-export const Query = {
+
+export const Query: Config["resolvers"] = {
   getPosts: async () => {
     return await getAllPosts();
   },
+
+  getPostById: async (_, id : string) => {
+    return await getPostById(id);
+  }
 };
 
-export const Mutation = {};
+export const Mutation: Config["resolvers"] = {
+
+  createPost: async (_, content: string) => {
```

Comment:
I think the function signature of this resolver is incorrect and I'm getting runtime errors. I think it needs to be:
```suggestion
  createPost: async (_, { content }: { content: string }) => {
```
Based on https://www.apollographql.com/docs/apollo-server/data/resolvers/#resolver-arguments, when the Apollo Server library calls the resolver method, it will pass the arguments send in the GraphQL request as an object that looks like `{ "id": "4" }` for example